### DISCUSSION TITLE
Cloudlog map showing incorrect location

### DISCUSSION THREAD

**Discussion by T012m3n7oR** [magicbug/Cloudlog#1004] [Q&A] [answered]
Hello Everyone,

I have ~15 logs in cloudlog that range from the end of last year ("recent" ticket, only just got into a club so I can use their radios, yay!) the map above the QSO log, doesn't appear to log the correct location.
![image](https://user-images.githubusercontent.com/48076977/116448080-b55e1e80-a826-11eb-8142-1dea79a11347.png)

I have one recent QSO that is the only QSO showing on the map, and it's showing in Wisconsin.  The Call sign is from VT, and the POTA site was in VT as well, I'm curious if there is something I can look at in the database or otherwise that could assist me in tracking down why it appears to be showing the wrong spot?

Thank you for your assistance!

**Comment by magicbug** [comment, marked as answer]
Cloudlog uses either gridsquare or dxcc county for map positions hence what your seeing.

**Reply by T012m3n7oR** [reply]
So if I understand you correctly, because the QSO didn't include a gridsquare, it used the DXCC of K (USA) and therefore is displaying it in Wisconsin?
It shows that this synced back from LOTW, and in that QSO on LOTW I do see a gridsquare.  Is this something that failed to pull back?  Does that value not pull back?  This did sync with the recommended cron job in the wiki.

I'm sorry for all of the questions I'm just trying to understand this so I can (hopefully) have a true map in the future.

Thank you for your assistance!
